src/Buckle/Buckle/CodeAnalysis/Binding/BoundScope.cs

```diff
@@ -95,21 +95,22 @@ internal sealed class BoundScope {
 
     internal void CopyInlines(BoundScope scope) {
         foreach (var inline in scope.GetDeclaredFunctions().Where(i => i.name.StartsWith("<$Inline")))
-            TryDeclareFunction(inline); // ignore failures, don't override higher level symbols
+            // Ignore failures, do not override higher level symbols
+            TryDeclareFunction(inline);
     }
 
-    internal ImmutableArray<FunctionSymbol> LookupOverloads(
```

Comment:
This function in design should only return functions, but because of how shadowing works (different things with the same name having different priorities) it needs to find the variables too in case a variable shadows all overloads